cmd/kubelet/app/server.go

```diff
@@ -944,7 +944,7 @@ func RunKubelet(kubeFlags *options.KubeletFlags, kubeCfg *kubeletconfiginternal.
 		kubeFlags.RegisterWithTaints,
 		kubeFlags.AllowedUnsafeSysctls,
 		kubeFlags.RemoteRuntimeEndpoint,
-		kubeFlags.RemoteImageEndpoint,
+		kubeCfg.RemoteImageEndpoint,
```

Comment:
i don't think you have to pass this like so, because the `kubeCfg` is already passed to `CreateAndInitKubelet()`.
just remove the `remoteImageEndpoint` argument from`CreateAndInitKubelet()`?